config/rubocop/1.9.1/rubocop-2.7.yml

```diff
@@ -16,6 +16,9 @@ AllCops:
 
 #################### Layout ##############################
 
+Layout/BeginEndAlignment:
+  EnforcedStyleAlignWith: begin
```

Comment:
I personally prefer [the default](https://docs.rubocop.org/rubocop/cops_layout.html#layoutbeginendalignment):

```
foo = begin
  ...
end
```

But we have code that does [the other style](https://github.com/bespokepost/spoke/blob/6bb5a12/app/admin/concerns/admin_policy_helpers.rb#L12-L16):

```
foo = begin
        ...
      end
```

I'd vote we remove this and conform to the default but wanted to point it out in case there's a preference

Reply:
I prefer the default as well.